src/scripts/main.js

```diff
@@ -1,3 +1,252 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('tbody');
+const titles = document.querySelector('thead');
+
+const form = document.createElement('form');
+const formSubmit = document.createElement('button');
+
+const inputs = ['Name', 'Position', 'Office', 'Age', 'Salary'];
+const employees = [];
+
+let currentOrder = 'asc';
+let currentTitle = '';
+let activeLine;
+let editing;
+
+const formatter = new Intl.NumberFormat('en-US');
+
+Array.from(table.children).forEach((el) => {
+  employees.push({
+    Name: el.children[0].textContent,
+    Position: el.children[1].textContent,
+    Office: el.children[2].textContent,
+    Age: el.children[3].textContent,
+    Salary: el.children[4].textContent.replace(',', '').replace('$', ''),
```

Comment:
better to use lowercase for object fields
```suggestion
    name: el.children[0].textContent,
    position: el.children[1].textContent,
    office: el.children[2].textContent,
    age: el.children[3].textContent,
    salary: el.children[4].textContent.replace(',', '').replace('$', ''),
```